### ISSUE TITLE
The ProductVersion property does not match the output from 'git --version'

### ISSUE THREAD

**Issue by RobertRad** [git-for-windows/git#1322]
- [X] I was not able to find an [open](https://github.com/git-for-windows/git/issues?q=is%3Aopen) or [closed](https://github.com/git-for-windows/git/issues?q=is%3Aclosed) issue matching what I'm seeing

The ProductVersion property shows `2.14.2.windows.2` while `git --version` shows `git version 2.14.2.windows.3`

The name of the installation file is `Git-2.14.2.3-32-bit.exe`

So the information in the ProductVersion property is not up-to-date.

### Details
- How to show the ProductVersion property
On Windows 7: Right-click on the file, Tab 'Details', Entry 'Product version'

 - Which version of Git for Windows are you using? Is it 32-bit or 64-bit?

```
$ git --version --build-options
git version 2.14.2.windows.3
built from commit: 924c19fc000d3759486b68bd73bc592fa02e367c
sizeof-long: 4
machine: x86
```

### Which files are affected
- git-bash.exe
- git-cmd.exe
- bin/bash.exe
- bin/git.exe
- bin/sh.exe
- cmd/git.exe
- cmd/git-gui.exe
- cmd/gitk.exe

**Comment by dscho** [comment]
With the 64-bit build, I get this:

![image](https://user-images.githubusercontent.com/127790/31577297-4d38cbee-b10c-11e7-9e36-7151d1bb18e9.png)

With the 32-bit portable Git (I do not want to run the 32-bit installer because I already have a 64-bit version installed) which is built from the very same set of files, I get this:

![image](https://user-images.githubusercontent.com/127790/31577346-f9d9909a-b10c-11e7-8f8b-58c678681333.png)

In other words: I cannot reproduce the issue here.

**Comment by RobertRad** [comment]
Seems to be a installation issue.

bin/git.exe is 2.14.2.windows.2
cmd/git.exe is 2.14.2.windows.2
mingw32/bin/git.exe is 2.14.2.windows.3
The latter is called from Git Bash.

After a fresh reinstall the files are alright.
The issue may have been caused by installing a 32bit version over a 64bit version.

So, fixed for me

**Comment by dscho** [comment]
Thanks for the clarification!
